portfolio/src/main/java/com/google/sps/storage/DatastoreCommentStorage.java

```diff
@@ -0,0 +1,83 @@
+package com.google.sps.storage;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.sps.storage.CommentStorage;
+import com.google.sps.data.Comment;
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class DatastoreCommentStorage implements CommentStorage {
+  private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
Lets use DI here as well... something like:
```java
  @Inject
  DatastoreCommentStorage(DatastoreService datastore) {
     this.datastore = datastore;
  }
```

Then your Datastore Binding module can still just provide a binding and a datastore:
```java
public class DatastoreBindingModule extends AbstractModule {

  @Override
  public void configure() {
    bind(CommentStorage.class).to(DatastoreCommentStorage.class).in(Singleton.class);
    // Add this line:
    bind(DatastoreService.class).toInstance(DatastoreServiceFactory.getDatastoreService()).in(Singleton.class);
  }
  // Or this method:
  @Provides
  @Singleton
  DatastoreService provideDatastoreService() {
    return DatastoreServiceFactory.getDatastoreService();
  }
}
```